dataset_interfaces/interface.py

```diff
@@ -294,6 +307,11 @@ def evaluate_correct(
         self, questions: List[str], responses: List[str], expected_answers: List[Any]
     ) -> Tuple[int, int, List[str]]:
         pass
+    
+    def evaluation_fn(self) -> Callable[[List[str], List[str], List[Any]], Tuple[float, int, List[str]]]:
+        def evaluator(questions: List[str], responses: List[str], expected_answers: List[Any]) -> Tuple[float, int, List[str]]:
+            return normalize_scores(self.evaluate_correct, questions, responses, expected_answers)
+        return evaluator
```

Comment:
Why is this here? The `evaluation_fn` is called as a part of the `TestExample`, not `DatasetInterface`

Reply:
This was added as the evaluation in the `migr-reeval.py` file uses the `DatasetInterface` class not the `TestExample` class